Solutions/Marain.UserNotifications.Storage.AzureStorage/Marain.UserNotifications.Storage.AzureStorage.csproj

```diff
@@ -15,11 +15,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Corvus.Azure.Storage.Tenancy" Version="2.0.7" />
-    <PackageReference Include="Endjin.RecommendedPractices" Version="1.2.0">
+    <PackageReference Include="Corvus.Storage.Azure.TableStorage.Tenancy" Version="3.2.0-PullRequest0409.5" />
+    <PackageReference Include="Corvus.Storage.Azure.BlobStorage.Tenancy" Version="3.2.0-PullRequest0409.5" />
+    <PackageReference Include="Endjin.RecommendedPractices.GitHub" Version="2.1.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
I forgot when we added this, but this should really be a floating reference too.